recipes/conf.rb

```diff
@@ -9,3 +9,14 @@
 when 'cis'
   auditd_conf_file 'cis.auditd'
 end
+
+template '/etc/audisp/plugins.d/syslog.conf' do
+  source 'audisp_syslog.conf.erb'
+  owner 'root'
+  group 'root'
+  mode '0640'
+  variables(
+    active: node['auditd']['syslog']
+  )
+  notifies :reload, 'service[auditd]', :delayed
+end
```

Comment:
In the future, it may make sense to create an `auditd_plugin` resource with properties that correspond to those configurable in `audispd` plugin config files (wrapping a `template` resource similar to this).